src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -392,6 +392,31 @@ def __str__(self):
         return f"{self.name}(data: {self.data})"
 
 
+@dataclass
+class RTVIObserverParams:
+    """
+    Parameters for configuring RTVI Observer behavior.
+
+    Attributes:
+        bot_llm_enabled (bool): Indicates if the bot's LLM messages are enabled.
+        bot_tts_enabled (bool): Indicates if the bot's TTS messages are enabled.
+        bot_speaking_enabled (bool): Indicates if the bot's started/stopped speaking messages are enabled.
+        user_llm_enabled (bool): Indicates if the user's LLM input messages are enabled.
+        user_speaking_enabled (bool): Indicates if the user's started/stopped speaking messages are enabled.
+        user_transcription_enabled (bool): Indicates if user's transcription messages are enabled.
+        metrics_enabled (bool): Indicates if metrics messages are enabled.
+    """
+
+    bot_llm_enabled: bool = True
+    bot_tts_enabled: bool = True
+    bot_speaking_enabled: bool = True
+    user_llm_enabled: bool = True
+    user_speaking_enabled: bool = True
+    user_transcription_enabled: bool = True
+    errors_enabled: bool = True
```

Comment:
nit: Missing the docs above for this one.

Reply:
yep, thank you!